edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/EdgeletFixture.cs

```diff
@@ -29,19 +29,22 @@ class ServiceHost
 
             ServiceHost()
             {
-                this.webHostTask = BuildWebHost(new string[0], DefaultPort).RunAsync(this.cancellationTokenSource.Token);
+                this.webHostTask = CreateHostBuilder(new string[0], DefaultPort).Build().RunAsync(this.cancellationTokenSource.Token);
                 this.Url = $"http://localhost:{DefaultPort}";
             }
 
             public static ServiceHost Instance { get; } = new ServiceHost();
 
             public string Url { get; }
 
-            static IWebHost BuildWebHost(string[] args, int port) =>
-                WebHost.CreateDefaultBuilder(args)
-                    .UseUrls($"http://*:{port}")
-                    .UseStartup<Startup>()
-                    .Build();
+            static IHostBuilder CreateHostBuilder(string[] args, int port) =>
+                Host.CreateDefaultBuilder(args)
```

Comment:
FYR https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#hostbuilder-replaces-webhostbuilder